src/NextGen/Framework/FormExtensions/ServiceProvider.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Give\NextGen\Framework\FormExtensions;
+
+use Give\Helpers\Hooks;
+use Give\NextGen\Framework\FormExtensions\Registrars\FormExtensionRegistrar;
+use Give\ServiceProviders\ServiceProvider as ServiceProviderInterface;
+
+/**
+ * @unreleased
+ */
+class ServiceProvider implements ServiceProviderInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function register()
+    {
+        give()->singleton(FormExtensionRegistrar::class, function () {
+            return new FormExtensionRegistrar();
+        });
```

Comment:
You don't need to use a callback unless you're doing something special when its instantiated:
```php
give()->singleton(FormExtensionRegistrar::class);
```